contracts/EthTranchePricing.sol

```diff
@@ -162,7 +154,7 @@ contract EthTranchePricing is PricingStrategy, Ownable {
   }
 
   function() payable {
-    throw; // No money on this contract
+    require(false); // No money on this contract
```

Comment:
The "payable" modifier could be removed with require(), resulting an empty function for clarity.

Reply:
Removed "payable" modifier and require() resulting in empty function as requested.